i18n/en.pot

```diff
@@ -180,3 +180,6 @@ msgstr ""
 
 msgid "Next"
 msgstr ""
+
+msgid "Use short names"
+msgstr ""
```

Comment:
Running `yarn localize` moves these new translation in the file, does it happen also for you?

Reply:
It does happen to me. I wasn't that familiar with the proper use of localize, so I missed it. I understand now that one should run it too when adding new texts. Fixed now, thanks!

Reply:
Ref: https://github.com/EyeSeeTea/d2-ui-components/commit/83b3c36285